README.md

```diff
@@ -12,6 +12,14 @@ Since version 8 Appium Java Client had several major changes, which might requir
 update your client code. Make sure to follow the [v7 to v8 Migration Guide](https://github.com/appium/java-client/blob/master/docs/v7-to-v8-migration-guide.md)
 in order to streamline the migration process.
 
+## Compatibility Matrix
+| Appium Java Client | Selenium client                            |
```

Comment:
maybe it also makes sense to add a link here to the documentation that describes how selenium version could be locked locally

Reply:
merged sections